### DISCUSSION TITLE
Enable Debug Mode Button for Users

### DISCUSSION THREAD

**Discussion by b-r-j** [tprouvot/Salesforce-Inspector-reloaded#691] [Ideas]
If we could have a button like "Enable Debug Mode," it would be really helpful because it is difficult and time-consuming for users to enable debug mode.  👍

**Comment by tprouvot** [comment]
Hi @b-r-j ,

You mean, like this ?
https://tprouvot.github.io/Salesforce-Inspector-reloaded/how-to/?h=enable#enable-debug-logs

😊

**Reply by b-r-j** [reply]
Not really, debug mode for users so that we can debug JavaScript code from Lightning components. :)

**Reply by tprouvot** [reply]
Ok understood 👍
I'll check that, I know that for enabling EPT we can add a parameter to the url, I have to check if it is the same for debug mode.

**Reply by b-r-j** [reply]
Cheers 🤩

**Comment by tprouvot** [comment]
Hi @b-r-j,
I found out how to enable it, now I have to work on the design since there is already many buttons on the users tab.

**Comment by tprouvot** [comment]
Will be part of the 1.26 and released in the beta next week

<img width="278" alt="image" src="https://github.com/user-attachments/assets/6599b811-e358-431c-9728-2fa5a861fb85" />

**Reply by b-r-j** [reply]
Great news 🥳
